NAMESPACE

```diff
@@ -15,6 +15,8 @@ export(find_ORFs_fa)
 export(find_in_frame_ORFs)
 export(floss)
 export(fpkm)
+export(fpkmRFP)
```

Comment:
I belive it should be just one function, fpkm that does exactly the same whether input Riboseq or RNAseq and normalization is optional. I would allow both normalize by length of ORF and length of Transcript.